src/js/common/schemaform/FormPage.jsx

```diff
@@ -138,7 +138,8 @@ class FormPage extends React.Component {
         touched: {},
         submitted: false,
         onEdit,
-        hideTitle
+        hideTitle,
+        pageTitle: pageConfig.title
```

Comment:
I think `pageConfig` was removed as the argument when I merged in changes from `master`. So it would need to be added back in to pass the currently failing check.